src/main/java/Commands.java

```diff
@@ -0,0 +1,155 @@
+public class Commands {
+    public enum Types {
+        TODO,
+        DEADLINE,
+        EVENT,
+        LIST,
+        MARK,
+        UNMARK,
+        BYE,
+        HELP,
+        DELETE,
+    }
+
+    public static Types checkType(String command) {
+        for(Types t : Types.values()) {
+            if (t.name().equalsIgnoreCase(command)) return t;
+        }
+        throw new RuntimeException();
+    }
+
+    public static void runCommand(Types t, String input, TaskList taskList) {
+        try {
+            if (input == null) {
+                switch (t) {
```

Comment:
for our module's special style norm indentation for switch and case is not recommended